src/main.py

```diff
@@ -42,7 +42,7 @@ def __init__(self, pID, condition) -> None:
         self.puzzleIndex = 0
 
         # Create Misty object
-        self.advice = Advice("192.168.2.216")
+        self.advice = Advice("192.168.2.216", condition)
```

Comment:
We probably want to have a variable to store the IP address, but this is fine for now